Source/control.cpp

```diff
@@ -468,7 +467,7 @@ void DrawSpellList(CelOutputBuffer out)
 					else
 						sprintf(tempstr, "%i Scrolls", v);
 					AddPanelString(tempstr, true);
-					break;
+				} break;

```

Comment:
I've never see a `break` outside of the scope like this. Is this standard?

Reply:
It's not something I came up with by self afaik. IMO It makes it clear that this is the final break and that the break is part of the switch and not something inside the block.

Personally I think that once your case grows so complex that it needs to define variables then it's time to create a sub-function for it. The good news here is that the style is also easily searchable compared to the one where it is on the inside.

Reply:
Fair enough. I was just honestly curious as I've really never seen it that way. Totally agree on the sub-function argument.

Reply:
We have accumulated 8 so far, but more might be revealed once the cleanups are done (5 were found in this PR).

Reply:
Yeah, recently I've adopted the same convention for personal projects. Pretty convenient I should say.